Dockerfile.full

```diff
@@ -0,0 +1,39 @@
+# Docker build file for mqttwarn.
+# Based on https://github.com/pfichtner/docker-mqttwarn.
+#
+# Invoke like:
+#
+#   docker build --tag=mqttwarn-local-full --file=Dockerfile.full .
+#
+FROM python:3.9-slim-buster
+
+# Install additional requirements
+# FIXME: Skip all packages needing compilation
+#RUN apt-get update && apt-get install --yes librrd-dev
+
+# Create /etc/mqttwarn
+RUN mkdir -p /etc/mqttwarn
+WORKDIR /etc/mqttwarn
+
+# Add user "mqttwarn"
+RUN groupadd -r mqttwarn && useradd -r -g mqttwarn mqttwarn
+RUN chown -R mqttwarn:mqttwarn /etc/mqttwarn
+
+# Install mqttwarn
+COPY . /src
+RUN pip install /src[all]
+
+# Convenience for @sumnerboy12
```

Comment:
No need for this, please feel free to remove.

Reply:
You mean, just the comment or the whole `pip install gpxpy`?

Reply:
The whole thing